tests/component.js

```diff
@@ -49,6 +49,30 @@ describe( 'CKEditor Component', () => {
 			expect( wrapper.html() ).to.not.be.empty;
 		} );
 
+		describe( 'when loading CDN script', () => {
+			let spy;
+
+			beforeEach( () => {
+				spy = sandbox.spy();
+
+				delete window.CKEDITOR;
+
+				setPropsForTestGroup( { scriptLoaded: spy } );
+			} );
+
+			afterEach( () => {
+				window.CKEDITOR = CKEDITOR;
+			} );
+
+			it( 'should notify about loaded namespace', () => {
+				expect( spy.calledOnce );
+
+				// We can't stub getEditorNamespace.scriptLoader because the logic
+				// responsible for notifying about loaded CDN is placed there.
+				// Let's just use simple duck typing instead.
+				expect( spy.calledWithMatch( namespace => !!namespace.version ) );
```

Comment:
Also here I think `expect()` lacks assertion. Also, I tried but didn't manage to understand to what `namespace => !!namespace.version` evaluates here?

Reply:
> Also here I think expect() lacks assertion

Yeah, I missed it somehow.

> Also, I tried but didn't manage to understand to what namespace => !!namespace.version evaluates here?

Since we are getting different `CKEDITOR` objects and they don't implement proper equality operator, it's just simple [duck typing](https://en.wikipedia.org/wiki/Duck_typing) (as noted in comment) checking if the object contains an expected property which should be enough to ensure that we are getting `CKEDITOR` object. However, after changes, it's no more required since we are changing the global config of the editor.